doc/guide/guide-bloch.rst

```diff
@@ -421,11 +421,13 @@ Directly Generating an Animation
 
 The code to directly generate an mp4 movie of the Qubit decay is as follows ::
 
+   import matplotlib
+   from qutip.bloch import axes3D_
    from matplotlib import pyplot, animation
-   from mpl_toolkits.mplot3d import Axes3D
 
    fig = pyplot.figure()
-   ax = Axes3D(fig,azim=-40,elev=30)
+
+   ax = axes3D_(fig, azim=-40, elev=30)
```

Comment:
I'd say there's no need to import the "fixed" `Axes3D` in the documentation.  Right now, just use whichever form works with matplotlib 3.3 - the documentation is currently built with matplotlib pinned to that.  Documentation doesn't need to build with _every_ version, it just needs to be repeatable, and I'd rather use the "core" version of `Axes3D` rather than a modified one in code that users will read.

(but @hodgestar if you want to weigh in differently, that's also fine - just mark this as resolved)

Reply:
Due to the way `axes3D_` works, there should never be any deprecation warnings (irrespective of the matplotlib version) that otherwise creep in while using matplotlib's `Axes3D` directly (given `matplotlib<3.4`). 

If it's okay to have deprecation warnings (for users having older versions of matplotlib) due to the code in the user guide, using `Axes3D` from `matplotlib` directly would be a good choice since it provides better readability. As a newcomer, I still would have been hesitant to see `axes3D_` imported from `qutip.bloch` in the user guide :)

Thanks!!

Any thoughts on this? @jakelishman @hodgestar

Reply:
Having read @jakelishman comment, I think we should remove the use of `axes3D_` from the documentation and maybe even rename it to `_axes3D`. We really don't want our users to be encouraged to use this internal helper function because then it will be harder to remove later.

@Yash-10 Thank you for putting up with the slightly back and forth reviewing! I think that are all the suggestions we're converging on a really nice solution.

Reply:
Thank you @hodgestar and @jakelishman ! Following your suggestions, I have renamed `axes3D_` to `_axes3D` and used matplotlib's `Axes3D` in the user guide to prevent any confusion there.

> @Yash-10 Thank you for putting up with the slightly back and forth reviewing! I think that after all the suggestions we're converging on a really nice solution.


Yes! Thanks for your guidance throughout.